examples/README.md

```diff
@@ -114,13 +114,15 @@ Toolkit team, partners, etc.) and are labeled with the community badge
 Blueprints that are still in development and less stable are also labeled with
 the experimental badge (![experimental-badge]).
 
-### [hpc-slurm.yaml] ![core-badge] ![experimental-badge]
+### [hpc-slurm.yaml] ![core-badge]
 
-> **Warning**: Requires additional dependencies **to be installed on the system deploying the infrastructure**.
+> **Warning**: The variables `enable_reconfigure`,
+> `enable_cleanup_compute`, and `enable_cleanup_subscriptions`, if set to
+> `true`, require additional dependencies **to be installed on the system deploying the infrastructure**.
 >
 > ```shell
 > # Install Python3 and run
-> pip3 install -r https://raw.githubusercontent.com/GoogleCloudPlatform/slurm-gcp/6.4.1/scripts/requirements.txt
+> pip3 install -r https://raw.githubusercontent.com/SchedMD/slurm-gcp/5.9.1/scripts/requirements.txt
```

Comment:
check on head of develop but I think we are now on 5.10.x

Reply:
done. it is 5.10.4